pebble-spring/pebble-spring5/src/main/java/com/mitchellbosecke/pebble/spring/reactive/PebbleReactiveView.java

```diff
@@ -55,6 +60,13 @@ protected Mono<Void> renderInternal(Map<String, Object> renderAttributes,
     return exchange.getResponse().writeWith(Flux.just(dataBuffer));
   }
 
+  private void addVariablesToModel(Map<String, Object> model, ServerWebExchange exchange) {
+    model.put(BEANS_VARIABLE_NAME, new Beans(this.getApplicationContext()));
+    model.put(REQUEST_VARIABLE_NAME, exchange.getRequest());
+    model.put(RESPONSE_VARIABLE_NAME, exchange.getResponse());
+    model.put(SESSION_VARIABLE_NAME, exchange.getSession());
```

Comment:
Hi @ebussieres , the return type of `exchange.getSession()` is `Mono<WebSession>`. We can't use it directly.